neurodsp/sim/cycles.py

```diff
@@ -255,9 +262,10 @@ def phase_shift_cycle(cycle, shift):
     ----------
     cycle : 1d array
         Cycle values to apply a rotation shift to.
-    shift : float
+    shift : float or str
```

Comment:
I think this this is a couple str options, and not any string, it should probably be
`shift : float or {'min', 'max'}`

I'm not 100% sure if that is the right convention, but it makes sense to me. If so this should be updated across all shift or phase argument descriptions.